ems_translations.csv

```diff
@@ -1,785 +1,787 @@
-Change;Verander;;;;
-Add an extra layer of security to your account.;Voeg een extra laag beveiliging toe aan je account.;;;;
-Enable two factor authentication;Schakel tweestapsverificatie in;;;;
-Disable two factor authentication;Schakel tweestapsverificatie uit;;;;
-Disable SSO;Schakel SSO uit;;;;
-Enable SSO;Schakel SSO in;;;;
-The following permissions have been granted to your account:;De volgende rechten zijn toegekend aan je account:;;;;
-Enter a reference;Voer een referentie in;;;;
-Past logins;Vorige logins;;;;
-You have logged into your account from the following devices:;Je hebt ingelogd op je account vanaf de volgende apparaten:;;;;
-Timestamp;Tijd;;;;
-Delete account;Verwijder account;;;;
-This will delete your user account. Note that once your account is deleted, it cannot be restored.;Dit verwijdert je gebruikersaccount. Let op: zodra je account is verwijderd, kan het niet worden hersteld.;;;;
-Delete your account;Verwijder je account;;;;
-Enter tags;Voer tags in;;;;
-This option disallows a user to apply a discount on a quotation line for this feature.;Deze optie zorgt er voor dat een gebruiker geen korting toe kan passen op een offerte regel voor deze feature.;;;;
-Warning;Waarschuwing;;;;
-Circular dependencies could take a minute to find, do you want continue?;Cirkel redeneringen vinden kan een minuut duren, wilt u doorgaan?;;;;
-Continue;Doorgaan;;;;
-Circular Dependency result;Cirkel redenering resultaten;;;;
-No Circular Dependencies found in the configuration model.;Geen cirkel redeneringen kunnen worden gevonden in het configuratie model.;;;;
-Find circular dependencies error;Vind cirkel redeneringen foutmelding;;;;
-Something went wrong while trying to find circular dependencies. Please try again later or contact support if it keeps occuring.;Er ging iets mis tijdens het vinden van cirkelredeneringen. Probeer het later opnieuw of contacteer support als het probleem zich blijft voordoen.;;;;
-Mobile phone number;Mobiel telefoonnummer;Numéro de téléphone mobile;Nummer des Mobiltelefons;Mobiltelefonnummer;Numer telefonu komórkowego
-Street name;Straatnaam;Nom de rue;Straßenname;Gatunamn;Nazwa ulicy
-This configuration is not possible. Your last change could not be saved.;Deze configuratie is niet mogelijk. Je laatste wijziging kon niet worden opgeslagen.;Cette configuration n'est pas possible. Votre dernière modification n'a pas pu être enregistrée.;Diese Konfiguration ist nicht möglich. Ihre letzte Änderung konnte nicht gespeichert werden.;;
-The issue could be caused by a change in the parent configuration.;De issue zou veroorzaakt kunnen zijn door een wijziging in een parent configuratie.;Le problème peut être causé par un changement dans la configuration parente.;Das Problem könnte durch eine Änderung in der übergeordneten Konfiguration verursacht werden.;;
-The description contains illegal characters.;;;;;
+en;nl;fr;de;sv;pl;no
+State;Provincie;RÃ©gion;Bundesland;Land;Kraj;Land 

```

Comment:
the encoding on the ems_translations seems wrong. It looks like ANSI encoding and should be UTF-8 encoding. Could you change it to that? If you need help doing that, let me know